app/src/main/java/net/gsantner/markor/frontend/textview/HighlightingEditor.java

```diff
@@ -178,6 +237,19 @@ public boolean setHighlightingEnabled(final boolean enable) {
         return prev;
     }
 
+    public boolean getLineNumbersEnabled() {
+        return _nuEnabled;
+    }
+
+    public boolean setLineNumbersEnabled(final boolean enable) {
+        final boolean prev = _hlEnabled;
```

Comment:
why does setLineNumbersEnabled return the old value of `_hlEnabled` instead of `_nuEnabled` ?

Since the returnvalue of `setLineNumbersEnabled(...)` is never used you can also make the method return void.